rust/ast-generator/patches/rust-analyzer.patch

```diff
@@ -55,3 +58,16 @@ index e7534582f2..8bc9237737 100644
      let mut res = AstSrc {
          tokens:
              "Whitespace Comment String ByteString CString IntNumber FloatNumber Char Byte Ident"
+diff --git a/xtask/src/codegen/grammar/ast_src.rs b/xtask/src/codegen/grammar/ast_src.rs
+index 9269d15423..babe7ca1bf 100644
+--- a/xtask/src/codegen/grammar/ast_src.rs
++++ b/xtask/src/codegen/grammar/ast_src.rs
+@@ -1,5 +1,8 @@
+ //! Defines input for code generation process.
+ 
++#![allow(clippy)]
++#![cfg_attr(any(), rustfmt::skip)]
++
+ use quote::ToTokens;
+ 
+ use crate::codegen::grammar::to_upper_snake_case;
```

Comment:
These added `use` statements look a bit odd.

Reply:
I think this might be the understandable confusion of looking at the diff of a diff: these lines are context lines in the new diff, added because I added the top-level attributes (which have a `+`, these lines don't)